packages/service/test/metrics.test.js

```diff
@@ -177,7 +177,7 @@ test('has /metrics endpoint on parent server', async (t) => {
   testPrometheusOutput(body)
 })
 
-test('support basic auth with metrics on parent server', async (t) => {
+test.only('support basic auth with metrics on parent server', async (t) => {
```

Comment:
```suggestion
test('support basic auth with metrics on parent server', async (t) => {
```

Reply:
@ivan-tymoshenko removed completely the `prefix`. The reason for it was to have a commit that we could apply also to v1.x (as exaplained in [this comment](https://github.com/platformatic/platformatic/pull/2875#discussion_r1690029392) and have the breaking change on main in another commit. We needed that because we cannot test with v2.0 (yet). 
However, this is not necessary anymore because I have created the PR also for v.1x (https://github.com/platformatic/platformatic/pull/2880)  with the same changes. PTAL on this and on the other for v1.x